DN1/test/runtests.jl

```diff
@@ -0,0 +1,35 @@
+using DN1
+using Test
+
+@testset "nakljucno izbrane tocke" begin
+  x = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+  y = [1.0, 3.0, 1.0, 2.0, 0.0, 6.0]
+  s = interpoliraj(x, y)
+  p1 = vrednost(s, 5)
+  p2 = vrednost(s, 4.9999)
+  p3 = vrednost(s, 0.0)
+  p4 = vrednost(s, 0.5)
+
+  @test p1 == 6.0
+  @test abs(p2 - 5.99) < 0.01
+  @test p3 == 1.0
+  @test p4 >= 1.0 && p4 <= 3.0
+end
+
+@testset "sinus" begin
+  x = range(0, stop=2 * pi, length=5)
+  y = sin.(x)
+  s = interpoliraj(x, y)
+
+  p1 = vrednost(s, 0)
```

Comment:
preverjate zgolj vrednosti v interpolacijskih točkah. Za to ne potrebujete funkcije `sin`.  Če bi znali oceniti napako za funkcijo `sin`, bi lahko to oceno uporabili za testiranje notranjih točk.